data/battle_scripts_2.s

```diff
@@ -14,16 +14,17 @@
 
 	.align 2
 gBattlescriptsForUsingItem::
-	.4byte BattleScript_PlayerUsesItem
-	.4byte BattleScript_OpponentUsesHealItem        @ AI_ITEM_FULL_RESTORE
-	.4byte BattleScript_OpponentUsesHealItem        @ AI_ITEM_HEAL_HP
-	.4byte BattleScript_OpponentUsesStatusCureItem  @ AI_ITEM_CURE_CONDITION
-	.4byte BattleScript_OpponentUsesXItem           @ AI_ITEM_X_STAT
-	.4byte BattleScript_OpponentUsesGuardSpec       @ AI_ITEM_GUARD_SPEC
-
-	.align 2
-gBattlescriptsForRunningByItem::
-	.4byte BattleScript_RunByUsingItem
+	.4byte BattleScript_ItemRestoreHP				@ EFFECT_ITEM_RESTORE_HP
+	.4byte BattleScript_ItemCureStatus	 			@ EFFECT_ITEM_CURE_STATUS
+	.4byte BattleScript_ItemHealAndCureStatus		@ EFFECT_ITEM_HEAL_AND_CURE_STATUS
+	.4byte BattleScript_ItemIncreaseStat 			@ EFFECT_ITEM_INCREASE_STAT
+	.4byte BattleScript_ItemSetMist					@ EFFECT_ITEM_SET_MIST
+	.4byte BattleScript_ItemSetFocusEnergy			@ EFFECT_ITEM_SET_FOCUS_ENERGY
+	.4byte BattleScript_RunByUsingItem				@ EFFECT_ITEM_ESCAPE
+	.4byte BattleScript_BallThrow					@ EFFECT_ITEM_THROW_BALL
+	.4byte BattleScript_ItemRestoreHP				@ EFFECT_ITEM_REVIVE
+	.4byte BattleScript_ItemRestorePP				@ EFFECT_ITEM_RESTORE_PP
+	.4byte BattleScript_ItemIncreaseAllStats		@ EFFECT_ITEM_INCREASE_ALL_STATS
```

Comment:
I think you've used tabs to align the `@`s here, but personally I'd recommend using spaces to align.
```suggestion
	.4byte BattleScript_ItemRestoreHP         @ EFFECT_ITEM_RESTORE_HP
	.4byte BattleScript_ItemCureStatus        @ EFFECT_ITEM_CURE_STATUS
	.4byte BattleScript_ItemHealAndCureStatus @ EFFECT_ITEM_HEAL_AND_CURE_STATUS
	.4byte BattleScript_ItemIncreaseStat      @ EFFECT_ITEM_INCREASE_STAT
	.4byte BattleScript_ItemSetMist           @ EFFECT_ITEM_SET_MIST
	.4byte BattleScript_ItemSetFocusEnergy    @ EFFECT_ITEM_SET_FOCUS_ENERGY
	.4byte BattleScript_RunByUsingItem        @ EFFECT_ITEM_ESCAPE
	.4byte BattleScript_BallThrow             @ EFFECT_ITEM_THROW_BALL
	.4byte BattleScript_ItemRestoreHP         @ EFFECT_ITEM_REVIVE
	.4byte BattleScript_ItemRestorePP         @ EFFECT_ITEM_RESTORE_PP
	.4byte BattleScript_ItemIncreaseAllStats  @ EFFECT_ITEM_INCREASE_ALL_STATS
```